### DISCUSSION TITLE
How do I disable "[GriefPrevention] Muted repeat message"?

### DISCUSSION THREAD

**Discussion by LinusLeonard** [GriefPrevention/GriefPrevention#2183] [Q&A] [answered]
If a player returns to my server, and three other players say e.g. "wb", I don't want player 2 and 3 to be muted, but I cannot for the life of me find where in the config.yml I can disable whatever it is that results in the server log message "[GriefPrevention] Muted repeat message".

**Comment by Jikoo** [comment, marked as answer]
That's part of the anti-spam feature. https://docs.griefprevention.com/configuration/#anti-spam-configuration

**Reply by LinusLeonard** [reply]
Oh I see. So in other words it's not possible to deactivate the muting of repeated messages without also deactivating the entire anti-spam feature?

**Reply by Jikoo** [reply]
Correct, the anti-spam isn't particularly configurable.
